core/src/main/java/org/apache/druid/java/util/common/Unsafe.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.java.util.common;
+
+import java.lang.reflect.Field;
+
+public class Unsafe
```

Comment:
Please don't use the same name as the JDK's class itself, use `UnsafeUtil` or `UnsafeHolder` or something like that.

Reply:
There are also usages of Unsafe in `HyperLogLogCollectorBenchmark`

Reply:
I can certainly rename the class. I don't really see the danger though since we will not be able to refer to sun.misc.Unsafe or jdk.internal.misc.Unsafe directly to be able to compile with JDK 9 and above.

Regarding HyperLogLogCollectorBenchmark, I plan to address it in a separate PR that also addresses Java 9 compatibility.